uber/models.py

```diff
@@ -1749,9 +1749,9 @@ class ArbitraryCharge(MagModel):
 
 
 class ApprovedEmail(MagModel):
-    subject = Column(UnicodeText)
+    ident = Column('subject', UnicodeText)  # TODO: rename column to "ident" in the database; will require a db migration
```

Comment:
Yeah, we have Alembic for migrations and they're pretty easy to create now. This is an interesting solution, though.